src/KiD_driver.jl

```diff
@@ -23,59 +27,46 @@ n_elem = 256
 t_ini = 0.0
```

Comment:
Another to-do: we should find out if this is total pressure or dry pressure, and that might solve remaining mismatches

Reply:
Opened an issue #21